src/Field.js

```diff
@@ -62,11 +62,11 @@ export default function Field(field) {
 		this.title = field.title;
 	}
 
-	if (field.min) {
+	if (typeof field.min === 'number') {
```

Comment:
Just realized we're already doing this in the assertion above, so we're kind of double-checking here, but I don't really see an issue with that :+1: